index.html

```diff
@@ -0,0 +1,130 @@
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  </head>
+  <body>
+    <div class="container">
+      <div class="sideNav">
+        <h1>JS documentation</h1>
+        <ul>
+          <li><a href="#intro">Introduction</a></li>
+          <li><a href="#"></a></li>
+          <li><a href="#what">What you should already know</a></li>
+          <li><a href="#js">Javascript or Java</a></li>
+          <li><a href="#hello">Hello world</a></li>
+          <li><a href="#variable">Variable</a></li>
+          <li><a href="#declare">Declaring variable</a></li>
+          <li><a href="#scope">Variable scope</a></li>
+          <li><a href="#global">Global variables</a></li>
+          <li><a href="#constant">Constants</a></li>
+          <li><a href="#data">Data types</a></li>
+          <li><a href="#condition">if...else statement</a></li>
+          <li><a href="#function">Function declarations</a></li>
+          <li><a href="#refrence">Refrence</a></li>
+        </ul>
+      </div>
+      <!-- end of side navigation -->
+      <div class="main">
+        <div class="intro">
+          <h2>Introduction</h2>
+          <p>
+            JavaScript is a cross-platform, object-oriented scripting language.
+            It is a small and lightweight language. Inside a host environment
+            (for example, a web browser), JavaScript can be connected to the
+            objects of its environment to provide programmatic control over
+            them.
+          </p>
+          <p>
+            JavaScript contains a standard library of objects, such as Array,
+            Date, and Math, and a core set of language elements such as
+            operators, control structures, and statements. Core JavaScript can
+            be extended for a variety of purposes by supplementing it with
+            additional objects; for example:
+          </p>
+          <ul>
+            <li>
+              Client-side JavaScript extends the core language by supplying
+              objects to control a browser and its Document Object Model (DOM).
+              For example, client-side extensions allow an application to place
+              elements on an HTML form and respond to user events such as mouse
+              clicks, form input, and page navigation.
+            </li>
+            <li>
+              Server-side JavaScript extends the core language by supplying
+              objects relevant to running JavaScript on a server. For example,
+              server-side extensions allow an application to communicate with a
+              database, provide continuity of information from one invocation to
+              another of the application, or perform file manipulations on a
+              server.
+            </li>
+          </ul>
+        </div>
+        <!-- end of intro -->
+        <div class="what">
```

Comment:
What is `what`?

Reply:
I think it is better to convey your message in a polite and inviting manner. There are many ways to rephrase this feedback to make it more polite, my suggestion is as follows: Nice work on naming the classes so far, but I suggest you rename your classes according to best HTML practices. For example, it should show the purpose of the section like you can name this as `info` instead of `what` which gives an exact idea of what is in the section.